src/test/scala/org/ergoplatform/utils/ErgoNodeViewHolderTestHelpers.scala

```diff
@@ -0,0 +1,144 @@
+package org.ergoplatform.utils
+
+import java.io.File
+
+import akka.actor.{ActorRef, ActorSystem}
+import akka.testkit.TestProbe
+import io.iohk.iodb.ByteArrayWrapper
+import org.ergoplatform.mining.DefaultFakePowScheme
+import org.ergoplatform.modifiers.history.Header
+import org.ergoplatform.modifiers.{ErgoFullBlock, ErgoPersistentModifier}
+import org.ergoplatform.nodeView.ErgoNodeViewRef
+import org.ergoplatform.nodeView.history.ErgoHistory
+import org.ergoplatform.nodeView.mempool.ErgoMemPool
+import org.ergoplatform.nodeView.state.{DigestState, ErgoState, StateType, UtxoState}
+import org.ergoplatform.nodeView.wallet.ErgoWallet
+import org.ergoplatform.settings.{Algos, ErgoSettings}
+import org.scalatest.BeforeAndAfterAll
+import scorex.core.ModifierId
+import scorex.core.NodeViewHolder.ReceivableMessages.GetDataFromCurrentView
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.FiniteDuration
+import scala.reflect.ClassTag
+
+trait ErgoNodeViewHolderTestHelpers extends ErgoPropertyTest with BeforeAndAfterAll with NoShrink {
```

Comment:
The name `ErgoNodeViewHolderTestHelpers` looks rather long

Reply:
I think it is ok. We have other long names in tests, and this one describes trait purpose